src/structure/options.ts

```diff
@@ -114,7 +117,7 @@ export class StructureOptions extends OptionsGroup {
             min: new HTMLOption('number', 0),
             max: new HTMLOption('number', 0),
             transform: new HTMLOption('string', 'linear'),
-            palette: new HTMLOption('string', 'BWR'),
+            palette: new HTMLOption('string', 'inferno'),
```

Comment:
I think we can still default to BWR for atom properties

Reply:
It is actually `bwr` now and not `BWR`